src/ui/sass/_base.scss

```diff
@@ -11,4 +11,8 @@
 
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+}
+
+.yxt-Answers-component--unmounted {
+  display: none !important;
```

Comment:
is there some way we could remove this `display: none` by not rendering the parent component until all of its children are rendered? I believe right now we render the parent container before all of the children are done rendering, though this might slow down the overall speed of non locator pages?

Reply:
We might need to clone the container node, attach children to the clone, and then replace the original when rendering is done?

Reply:
I'm not sure that is possible without a larger refactor. Currently handlebars puts the divs for the vertical result cards on the DOM, so we would need to avoid that. That would cause an issue because subcomponents are built by querying the DOM and reading the data on the child divs.

Reply:
that's fair, touching addChild() would be pretty annoying

Reply:
I think this is fine since it's an experimental feature